projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -995,12 +995,7 @@ public function get_filtered_connection_data( $selected_post_id = null ) {
 				$connection_list[] = array_merge(
 					$connection,
 					array(
-						'enabled'    => $enabled,
-						// Deprecated fields.
-						'id'         => $connection_id,
-						'unique_id'  => $unique_id,
-						'done'       => $done,
-						'toggleable' => $toggleable,
```

Comment:
I don't think toggleable is used anywhere anymore, but I have the same question here as before. Could there be any conflicts with multiple plugin versions?

Reply:
No conflicts because this is all inside publicize package which will always be the latest version.